<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rechargeBalanceWindow</class>
 <widget class="QDialog" name="rechargeBalanceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="amountDisplay">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="amountTitle">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="depositButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawButton">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>220</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QLabel" name="rechargeBalanceTitle">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Recharge Balance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="balanceStatusDisplay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentBalanceTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentBalanceDisplay">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showCurrentButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Current</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
